### DISCUSSION TITLE
What is the right image for Raspberry 4B/8GB

### DISCUSSION THREAD

**Discussion by GaupeLnx** [opencardev/crankshaft#571] [Q&A]
I am trying to install Crankshaft on a Raspberry Pi 4B/8GB
I Download both 2022-02-04-crankshaft-ng-496df4a-pi2.zip   and 2022-02-04-crankshaft-ng-496df4a.zip

I have a 1920x1080 Computer monitor attached to hdmi0 
and a keyboard and mouse to USB ports

The  2022-02-04-crankshaft-ng-496df4a.zip    does not start  openauto neither in X11 or EGL mode  
while when i flash the 2022-02-04-crankshaft-ng-496df4a-pi2.zip    its starts fine.

Am i mistaken in that the image from the 2022-02-04-crankshaft-ng-496df4a.zip  file should be  the right image for the raspi 4B ?
and not the image from the  2022-02-04-crankshaft-ng-496df4a-pi2.zip file ?

**Comment by Geekyadz** [comment]
I would try Crankshaft OpenCarOS NG Alpha-7.4 2021-09-22

**Reply by matt2005** [reply]
Try the latest release, a naming issue has been resolved.
